gruntfile.js

```diff
@@ -54,10 +55,9 @@ module.exports = function (grunt) {
     'if:linux386BinaryNotExist',
     'html2js',
     'useminPrepare:release',
-    'recess:build',
+    'postcss:build',
```

Comment:
The `postcss:build` task should be right after the `concat` task in all the `release-*` tasks.